### DISCUSSION TITLE
In-class Thinking Cap 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#24] [Assignments]
Respond to ANY or all of the prompts below in 2-3 sentences:

### 4. How do you seek to be peer and/or mentor for civil society?

You do not need to respond to any students.

**Comment by zacharypakin** [comment]
Through stating my problem, I was able to list out what we have done already to approach it, which made it seem like a less daunting task to finish. My partner shared similar problems, which made me feel as if my challenges are more easily overcome and that I am not alone. My first thought when hearing my partner's problem was to give advice, as this is a default response for me, but I also recognize that it is important to allow people to share their problems without suggesting how they can fix them. I think this exercise was useful to give me more experience being a listener rather than a fixer.

**Comment by catherinehhu** [comment]
I definitely think that stating my problem out loud helped because it required that I verbalize my thoughts and convey them in a way that someone else would understand. It allowed me to zoom out of the very specific ways I've been thinking and consider it from a big picture instead.

**Comment by emai14** [comment]
Stating my problem made it feel a lot more simple, and less overwhelming. My partner helped lead me to some good conclusions through asking me more about the problem.

**Comment by joanneluong** [comment]
Talking out loud about my problems helped me figure out exactly what it was that I was stuck on. I feel that as I spoke, I was clarifying myself to get to the exact problem that I was trying to get at, whereas in my head it was kind of just a jumble of thoughts. I really appreciated having the time to do this activity!

**Comment by czhangy** [comment]
I think that stating the problem our group was having helped force me to take a look at the bigger picture of the project and showed me that the issue may not be as problematic as I believed it was. My partner helped me think about if our solution was actually pushing us in the correct direction vs. just making a decision and running with it.

**Comment by paige-lee** [comment]
No, it was not difficult to avoid giving advice. I think this is because in general, I am often a listener and won't offer advice unless the other person asks for it

**Comment by scaucla** [comment]
I struggled to verbalize my problem to someone who is not in my group, however when I was finally able to articulate the problem it made the issue less daunting for me. My partner helped me refocus the problem to the core idea of this project.

**Comment by jamesfalkenroth** [comment]
My partner brought up some interesting ideas that I had not considered. The feature I am having trouble with is a Spanish translation in our survey. My partner asked about how important the feature was to our project and who was working on it. This helped me clarify the importance of having the feature, which would empower more voices in Los Angeles, a city with a large Latino population. It also helped me consider how I would work on the problem moving forward and how I would involve my group.

**Comment by enyi-emuka** [comment]
I think verbally constructing my difficulties into words and saying them to a peer allowed for them to be less of an abstract emotional object ("I am afraid for my project because xyz") and more neutral with actionable steps to deal with it, essentially my problem was reconfigured into it's most simple form. It was hard to not give advice because I am a person that is oriented towards fixing things immediately, asking follow up questions made me consider the context and the desires of my partner and their group.

**Comment by chrysanthemma** [comment]
Stating our problem out loud does force me to synthesize and process it in a more concrete way. My partner didn't necessarily bring up anything that I hadn't considered, but it was still a nice conversation nonetheless. I guess it was somewhat difficult in that giving advice seems the most natural. I seek to be a peer and mentor for civil society by continuing to learn from others and taking a stance of humility in that way. I hope to continue to grow in empathy and an other-centered-ness rather than being wrapped up in my own ideas.

**Comment by vickydelk** [comment]
I found the act of stating my problem out loud to be extremely helpful as it forced me to hear it in a more objective way than I was previously viewing it. I also think that summarizing a problem into words helps break it down and makes it seem like an easier task to accomplice rather than less concrete thoughts.  My parter did not bring up anything that I had not considered but the act of talking it through helped me to think up a new solution.

**Comment by danigold1020** [comment]
Yes, stating the problem out loud made me realize that our survey participants won't just share their stories or their experiences, but also those who may not have experienced oil spills first-hand may still want to express their concerns regarding their potential location near an oil source or future oil spills in their community. This would allow our mapplication to become a platform that sparks further conversation and may inspire action or at least comfort communities concerned about oil spills. It was not difficult to avoid giving advice since we came into the discussion with a curious mind to listen to our partner, and I came up with more follow-up questions than suggestions for my partner.

**Comment by cassiuscarandang** [comment]
I think it wasn’t difficult to avoid getting advice because I myself asked for any constructive criticism or advice in general that can help better my project. I was able to receive advice and follow up questions about revising my group’s memo and survey questions.

**Comment by SageLuong** [comment]
Simply stating my problem out-loud allowed my brain to reason with my fear of sourcing opinions outside of UCLA, especially Westwood businesses. It also allowed me to start thinking of other alternatives and ideas of how to tackle my fear. I was able to ask some clarifying questions for my mentees situation to see what they want from their project, and I tried to focus more on reassuring and leading them to contemplate on what they want to represent by focusing on first generation Filipino students. It was still hard to not offer advice immediately since I'm used to trying to solve a problem immediately and quickly.

**Comment by DylanJWan** [comment]
Stating the problem out-loud didn't necessarily give more insight but it did help me to articulate it. This made the problem feel less overwhelming because it was no longer so nebulous. The following questions that my partner asked made the problem even less intimidating, as she hinted at resources I could look into.

**Comment by ariscorpus** [comment]
My partner actually did raise a valuable point and idea that I didn't consider. They understood the problems that I presented and was able to come up with an idea that could be very valuable to me and my group mates. It was also something that me and my group mates didn't consider before so it was a welcomed idea.

**Comment by cassidytu** [comment]
The act of explaining my problem to an "outsider" perspective allowed for me to take a step back and reflect/summarize what we have done in our project. I was able to focus on what was a vital detail to include and my partner asked questions about the significance of certain things, which reemphasized the important things to focus on within the scope of our project.

**Comment by kmtsujimoto** [comment]
Stating my problem helped to clarify some of the ideas I had been considering but wasn't entirely sure how to execute. My partner's questions helped me to reevaluate some of the goals we had initially considered for our project, but had not properly evaluated recently understanding the skills we have acquired and the feasibility of different tasks within our skill set and timeline. It was difficult to focus more on follow-up questions than giving direct advice, but I appreciated the difference given the benefits of my partner's questions for my own thought process. This exercise helped me better understand the difference of providing solutions for communities and providing tools that community can use to address their problems, which I hope to apply through our project.

**Comment by angelicaalacantar** [comment]
Articulating my problems out loud is very helpful in that it forces me to identify what my problems are and to get more specific about it and how to address it. Receiving advice has also been very helpful as it provides us with another perspective that can be refreshing and reassuring that the problem wasn't too difficult and can be solved.

**Comment by qatle** [comment]
Explaining my group project to someone else who doesn't know about it really helps with redefine the goals and simplifying the problems. I also got a really good follow-up and feedback that I thought was useful to apply to the project

**Comment by stellakangj** [comment]
Overall, I felt that communicating the problem with my partner faced in their project also helped me gain new insights for my group project. After sharing the most recent problem my group project had, I felt that my own understanding of the problem increased when I heard my partner re-summarized the problem in their own words. Hearing the problem from the thought process of my partner helped me have a deeper understanding of the problem.

**Comment by maizaha** [comment]
I felt like explaining my group project out loud to fresh ears helped me work through my own jumbled thoughts on the project, as I was able to systematically walk my partner through the group project and the reasoning behind each of the decisions I've made. This helped me feel more sure about the direction I'm taking with the group project and how I've been approaching the problem thus far.

**Comment by mandymwlai** [comment]
Stating my problem out loud allowed me to verbalize the issues we've faced in the project. It was helpful in defining the scope of the problem and make it seem less daunting. I do admit it originally seemed counterintuitive to avoid giving advice since I was so used to doing that during peer review sessions. I believe I've developed a problem-solving, solution-oriented mindset towards problems through my years of education.

**Comment by shintuzd** [comment]
Saying my problem out loud helped me realize that I wasn’t struggling with the issue, but rather looking for the code to actualize my vision. It demonstrated that my ideas were there, but I was trying to find the tools to lay it out rather than figure out what to create in the first place. When flipped around, it was difficult to not follow-up the conversation with advice since that is what we’re used to doing.

**Comment by JonathanGLopez** [comment]
Stating my problem aloud brought new insights since it outlined the steps needed to complete the project and overcome the problem. It was not difficult to get advice since it was a close friend giving feedback and hearing me out. This class has changed the way I view technology through the lens of data justice and design justice. I now understand how to use the tools I have gained as a UCLA student for civil society, but had made me reevaluate how I understand research and technology.

**Comment by trufflelover** [comment]
Stating my problem outline was very helpful because it helped me organize my problems better off of my head. I want to become a mentor for the civil society by promoting justice in the digital realm.  I want to be able to shed light on the injustice that exists specifically in algorithm and promote data justice.

**Comment by amandatlxu** [comment]
I seek to be a peer and/or mentor for civil society by sharing and advocating for things such as design justice and data justice in my future endeavors. By doing so, I hope to increase accessibility and ethicality among technical spaces. This will in turn help combat injustices in civil society.
